src/well-known-headers.json

```diff
@@ -60,22 +45,14 @@
   "concealed-auth-export",
   "configuration-context",
   "connection",
-  "content-base",
   "content-digest",
   "content-disposition",
   "content-encoding",
   "content-id",
-  "content-language",
   "content-length",
-  "content-location",
   "content-md5",
-  "content-range",
```

Comment:
this one is multipurpose
https://developer.mozilla.org/en-US/docs/Glossary/Content_header